### ISSUE TITLE
Harmonic noise in MB Melgan if not trimming silence in training

### ISSUE THREAD

**Issue by fervmsanas** [kan-bayashi/ParallelWaveGAN#294]
![Screen Shot 2021-08-19 at 3 05 28 PM](https://user-images.githubusercontent.com/84736755/130150602-855b68cf-8390-495c-8145-5c0c5682f1c6.png)

Hi, I got some harmonic noise at silence segments  (top plot) if I set the silence trimming option to False for MB Melgan training. It does not happen if I set this to True (bottom plot)  and I keep the same training conditions. I have been trying different configurations and this is the only case  when I get this issue. Any thoughts or experience on  this?, thank you in advance.

**Comment by kan-bayashi** [comment]
This is an interesting observations.
I have never seen this kind of phenomenon but in my experiences trimming the silence part as much as possible leads the improvement of the quality. 
Not directly related to this issue, but hifigan discriminator is very strong so I’m curious the combination mb melgan G + hifigan D.

**Comment by ZDisket** [comment]
@kan-bayashi 
> but hifigan discriminator is very strong so I’m curious the combination mb melgan G + hifigan D.

I've been using that combination for the past few months and it seems to help a lot for making the generator finetuneable and increasing multispeaker performance. With hifigan discriminator and a learning rate decrease I can adapt mb-melgan generator within 10 to 30k steps to new voice. This is [my config](https://github.com/ZDisket/TensorflowTTS/blob/lju/examples/multiband_melgan_hf/conf/multiband_melgan_hf.lju.v1.yml), I can share samples.

**Comment by yt605155624** [comment]
mark
